bundles/org.openhab.binding.resol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="resol"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="device">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="vbuslan"/>
+		</supported-bridge-type-refs>
+
+		<label>Resol Device</label>
+		<description>Solar or system controller (or any other real device on the VBus) from Resol.</description>
+	</thing-type>
+
+	<thing-type id="emulatedEM">
```

Comment:
What do you mean by emulated?

Reply:
Normally an "EM" is a physical device which extends the solar controller by some sensor and relay channels and it is connected to the VBUS. This thing type allow openHAB to act as such a slave device on the VBUS and transmit sensor inputs and read relay output states to and from the solar controller. If a physical EM device is attached openHAB can only read the relay output and the sensor inputs measured by it but not transmit temperature values which will be handled by the solar controller.

Reply:
normally an EM device is a physical unit which extends the solar controller by 6 more sensor inputs and 5 relay outputs. If those are transmitting data on the VBUS they are discovered and openHAB is able to read the sensor inputs and relay output states.

The target of emulating an EM is to make openHAB a slave on the VBUS to be able to actively write sensor data which will then be used as "module sensors" in the solar controller, and the solar controller does not know that those values are injected by openHAB but will "see" them as another EM.